pom.xml

```diff
@@ -195,9 +174,9 @@
       <version>19.0</version>
     </dependency>
     <dependency>
-      <groupId>cryptix</groupId>
-      <artifactId>cryptix-jce</artifactId>
-      <version>na</version>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcprov-jdk15on</artifactId>
```

Comment:
Adjust THIRDPARTYLICENSE.txt

Reply:
BC's license was already there (since BC is OpenSAML's dependency):
(Bouncy Castle Licence) Bouncy Castle Provider (org.bouncycastle:bcprov-jdk15on:1.51)

- removed Cryptix license
- added JUnit's license